ts/client/src/client.ts

```diff
@@ -1141,18 +1193,49 @@ export class MangoClient {
     orderType: Serum3OrderType,
     clientOrderId: number,
     limit: number,
-  ): Promise<TransactionInstruction> {
+  ): Promise<TransactionInstruction[]> {
+    const txes: TransactionInstruction[] = [];
     const serum3Market = group.serum3MarketsMapByExternal.get(
       externalMarketPk.toBase58(),
     )!;
+    let openOrdersPk;
+    let additionalAccounts: AdditionalHealthAccounts | undefined = undefined;
     if (!mangoAccount.getSerum3Account(serum3Market.marketIndex)) {
-      await this.serum3CreateOpenOrders(
+      const ix = await this.serum3CreateOpenOrdersIx(
         group,
         mangoAccount,
         serum3Market.serumMarketExternal,
       );
-      await mangoAccount.reload(this);
+
+      const [openOrderPublicKey] = await PublicKey.findProgramAddress(
+        [
+          Buffer.from('Serum3OO'),
+          mangoAccount.publicKey.toBuffer(),
+          serum3Market.publicKey.toBuffer(),
+        ],
+        this.program.programId,
+      );
+      const baseBank = group.getFirstBankByTokenIndex(
```

Comment:
Im using baseTokenIndex in here few lines lower we have this => 

```
const payerTokenIndex = ((): TokenIndex => {
      if (side == Serum3Side.bid) {
        return serum3Market.quoteTokenIndex;
      } else {
        return serum3Market.baseTokenIndex;
      }
    })();
```

should i use only basetokenIndex or take the index form this function ?

Reply:
fixed